docs/tableBeamGroup1.adoc

```diff
@@ -154,14 +156,22 @@ e|Variables | |
  |{var}float detected_bottom_range(ping_time, beam) |O |Range from the transducer face where the bottom detection criteria were encountered for the amplitude or the phase of the backscattered echoes. The range of the bottom at index bottom_index with a monostatic transducer and if a constant sound speed is applied is given by detected_bottom_range= sound_speed_at_transducer*(blanking_interval+bottom_index*sample_interval - sample_time_offset)/2.
  3+|{attr}:long_name = "Detected range of the bottom" 
  3+|{attr}:units = "m" 
- 3+|{attr}:valid_min = "0.0" 
+ 3+|{attr}:valid_min = "0.0"
+
+ |{var}float transducer_impedance(ping_time, subbeam) |MA |Impedance of the transducer. This is the impedance of the load over which the transceiver measures voltage on the transducer subbeam. Necessary for conversion equation type 4.
```

Comment:
Impedance can be a complex number. Do we ever expect to have complex impedances for the transducer and receiver?

Reply:
Indeed, and it could then be used to store the complex impedance that would be calculated from the transmit time of the pulse with EK80, we however have currently inconsistency in the encoding of the samples for this first part of the ping in SonarNetcdf format if we simply copy the raw EK80 samples.
We should get back to these two points later.